lib/developer_portal/app/views/developer_portal/shared/_swagger_ui.html.liquid

```diff
@@ -11,15 +11,18 @@
   (function () {
     var url = "{{provider.api_specs.first.url}}";
 
-    SwaggerUI({
+    window.swaggerUi = new SwaggerUi({
```

Comment:
why changing this back to version 2?

Reply:
TLDR: cecause it is not being used in v3.
Some cukes have been failing since `swagger_ui` is still being used for v2. For v3 we updated the parent document directly and it is not using `_swagger_ui.html.liquid` anymore.